media_daemon/auto_skills.py

```diff
@@ -1,26 +1,28 @@
 import os
 from glob import glob
 import shutil
-from subprocess import check_output #nosec #pylint-disable type: ignore
+import subprocess #nosec #pylint-disable type: ignore
+from mount import *
 
 # To get media_daemon folder
 media_daemon_folder = os.path.dirname(os.path.abspath(__file__))
 base_folder = os.path.dirname(media_daemon_folder)
 server_skill_folder = os.path.join(base_folder, 'susi_server/susi_server/data/generic_skills/media_discovery')
 server_settings_folder = os.path.join(base_folder, 'susi_server/susi_server/data/settings')
+server_restart_folder = os.path.join(base_folder, 'susi_server/susi_server/bin/restart.sh')
 
 def make_skill(): # pylint-enable
+    devices = list_media_devices()
+    path = get_media_path(devices[0])
+    subprocess.call(['udisks','mount',path])  #nosec #pylint-disable type: ignore
     name_of_usb = get_mount_points()
-    print(type(name_of_usb))
-    print(name_of_usb[0])
-    x = name_of_usb[0]
-    os.chdir('{}'.format(x[1]))
-    USB = name_of_usb[0]
+    USB = name_of_usb[1]
+    os.chdir(str(USB))
```

Comment:
Stop moving around with `os.chdir`.

Reply:
@stealthanthrax Read: https://github.com/fossasia/susi_linux/issues/237

Didn't you do on such topic already?